client/lib/plans/index.js

```diff
@@ -220,19 +220,30 @@ export function plansLink( url, site, intervalType ) {
 
 export function applyTestFiltersToPlansList( planName ) {
 	const filteredPlanConstantObj = { ...plansList[ planName ] };
-	let filteredPlanFeaturesConstantObj = pick( featuresList, plansList[ planName ].getFeatures() );
+	let filteredPlanFeaturesConstantList = plansList[ planName ].getFeatures();
 
 	const removeDisabledFeatures = () => {
 		if ( ! isWordadsInstantActivationEnabled() ) {
-			filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
-				( value, key ) => key !== FEATURE_WORDADS_INSTANT
+			filteredPlanFeaturesConstantList = filter( filteredPlanFeaturesConstantList,
+				( value ) => value !== FEATURE_WORDADS_INSTANT
 			);
 		}
+
 		if ( ! isGoogleVouchersEnabled() ) {
-			filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
-				( value, key ) => key !== FEATURE_GOOGLE_AD_CREDITS
+			filteredPlanFeaturesConstantList = filter( filteredPlanFeaturesConstantList,
+				( value ) => value !== FEATURE_GOOGLE_AD_VOUCHERS_100 &&
+					value !== FEATURE_GOOGLE_WORDADS_AD_VOUCHERS_200
 			);
 		}
+
+		if ( ! isWordpressAdCreditsEnabled() ) {
+			const wordpressAdCreditsFeatureIndex = filteredPlanFeaturesConstantList.
+				indexOf( FEATURE_GOOGLE_WORDADS_AD_VOUCHERS_200 );
+
+			if ( wordpressAdCreditsFeatureIndex !== -1 ) {
+				filteredPlanFeaturesConstantList[ wordpressAdCreditsFeatureIndex ] = FEATURE_GOOGLE_AD_VOUCHERS_100;
```

Comment:
But you are updating a description. You're replacing "$100 Advertising Voucher" with "$200 Advertising Voucher" and we've been calling that content (which represents a plan's feature in its column) a description. I think you're calling that a title in your comment and you're calling what was referred to as "infoPopup" in the prior code as a "description". I don't mind if we change the terminology, but we should be consistent. Part of the reason I split all of this up this way is because I found it very difficult to tell what `filteredPlanConstantObj` and `filteredPlanFeaturesConstantList` were actually referring to.